src/storage/conversion/ErrorToJsonConverter.ts

```diff
@@ -22,6 +23,11 @@ export class ErrorToJsonConverter extends BaseTypedRepresentationConverter {
       message: error.message,
     };
 
+    // OAuth errors responses require additional fields
+    if (OAuthHttpError.isInstance(error)) {
+      Object.assign(result, error.out);
+    }
+
```

Comment:
Can we do that more neatly? So have a more meaningful name that's not `.out` and then check for that field?

```suggestion
    // OAuth errors responses require additional fields
    if (error.details) {
      Object.assign(result, error.details);
    }

```